Opcodes/emugens/emugens.c

```diff
@@ -2819,8 +2819,12 @@ int32_t printsk_init(CSOUND *csound, PRINTLN *p) {
                                              p->strseg.data, maxSegmentSize);
         p->strseg.size = maxSegmentSize;
         p->allocatedBuf = 1;
-        csound->RegisterResetCallback(csound, p,
-                                      (int32_t(*)(CSOUND*, void*))(println_reset));
+        /* VL 22.03.25 this callback is causing access-after-free as 
```

Comment:
Think if this is going to get commented out, might as well delete this block of commented code and remove println_reset altogether.

Reply:
I commented it out because it's not our (core devs) code. The author might want to re-use it or whatever without having to go into the git history.

Reply:
@gesellkammer could you take a look at this and comment on whether it's fine to remove? Seems to me that everything that is in println_reset is already cleared by standard memory cleaning in csoundReset and thus the problem occured in trying to free a second time.

Reply:
yes, it is fine to remove it

Reply:
Thanks for confirming, I'll go ahead and remove then merge this PR.